internal/app/handlers/handlers.go

```diff
@@ -38,14 +39,16 @@ func PostHandler(res http.ResponseWriter, req *http.Request, storage *storages.S
 	u, err := io.ReadAll(req.Body)
 
 	if err != nil {
-		http.Error(res, "Something went wrong!", http.StatusInternalServerError)
+		log.Println(errorText)
```

Comment:
Идея логирования заключается в том, что нам как разработчикам можно посмотреть, что произошло, когда что-то пошло не так. То есть в лог нам надо поместить текст ошибки, а пользователю вернуть какое-то общее сообщение.

То есть что-то такое:

```suggestion
		log.Printf("failed to read the request body: %v", err)
		http.Error(res, "", http.StatusInternalServerError)
```